src/cloud/components/Editor/index.tsx

```diff
@@ -421,8 +429,21 @@ const Editor = ({
         })
       }
     })
-    editor.on(
-      'cursorActivity',
+    editor.on('cursorActivity', (cm) => {
+      const cursorColumn = cm.getCursor().ch
+      const currentLine = cm.getLine(cm.getCursor().line) || ''
+      if (
+        !cm.state.completeActive &&
+        // user has started with '```x' and is waiting for options
+        currentLine.startsWith('```') &&
+        currentLine.length >= 4 &&
+        cursorColumn >= 3
+      ) {
+        // todo: [komediruzecki-2022-03-27] See how to handle no suggestions
```

Comment:
Don't show the hint popup if nothing match.

Reply:
Actually, it won't be shown, I mean the API will be called but an empty suggestion list will be returned (actually null returned from the underlying suggestion hint function) - so nothing will be shown - but in a general sense, we could add this check